regression/ebmc/SVA-LTL/lasso1-counterexample.desc

```diff
@@ -0,0 +1,7 @@
+KNOWNBUG
+lasso1.sv
+--bound 10 --trace
+^EXIT=10$
+^SIGNAL=0$
+^\[top\.property\.p0\] .* FAILURE$
+--
```

Comment:
It would be great to have some explanation (after another `--` line) of the problem that renders this test (and also the other one in this PR) a "KNOWNBUG."

Reply:
done